action.yml

```diff
@@ -203,15 +213,15 @@ runs:
         ls -l '${{ steps.cerbero-config.outputs.cerbero-home }}'
         echo "::endgroup::"
 
-    - id: bootstrap-cerbero
+    - id: cerbero-build
```

Comment:
`bootstrap-cerbero` may not be the best ID, but this step doesn't really "build Cerbero," it does everything in the [`cerebero_deps_script()` from `cerbero_setup.sh`](https://github.com/blinemedical/cerbero/blob/0748e1cbc748b4bc773d7c3bf0aaf3f10c505d75/ci/cerbero_setup.sh#L112-L150) up until uploading the cache.

Reply:
I struggled with even this name as much as the original after the dozenth build failure, TBH.  So much "bootstrapping" happening throughout in multiple GHA steps to the point that even telling cerbero to `build` ...it still just bootstraps some more until `package` is run in a different GHA step.  I went with `build` since that was the penultimate set of actions passed to `cerbero-uninstalled`, which is itself a really odd name for a script.

Reply:
That said, what is your suggestion given how overloaded "bootstrap" is in this context and how "build" doesn't fully build?

Reply:
Maybe `build-gstreamer-deps`? Should probably update the `# Bootstrap Cerbero` commend below as well.

Reply:
Done.